sourcemash/frontend/static/js/views/feed_view.js

```diff
@@ -8,8 +8,13 @@ Sourcemash.Views.FeedView = Sourcemash.Views.ItemsView.extend({
     toggleSubscribed: function() {
         if (this.model.get('subscribed')) {
             this.model.save({'subscribed': false}, {success: this.toast});
+            
+            mixpanel.track("Unsubscribed", { "Feed Title": this.model.get('title') })
```

Comment:
Right now there's only one way to unsubscribe (the toggle button). Subscribe can happen in many ways and I want to keep track of what prompted the subscription (that's the motivation behind our film project). 

Alex Gerstein
gerstein.alex@gmail.com




On Sat, Feb 28, 2015 at 3:50 PM -0800, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/static/js/views/feed_view.js:
> @@ -8,8 +8,13 @@ Sourcemash.Views.FeedView = Sourcemash.Views.ItemsView.extend({
>      toggleSubscribed: function() {
>          if (this.model.get('subscribed')) {
>              this.model.save({'subscribed': false}, {success: this.toast});
> +            
> +            mixpanel.track("Unsubscribed", { "Feed Title": this.model.get('title') })



Not tracking "Source" here but tracking it below? 



—
Reply to this email directly or view it on GitHub.